src/contracts/service/BondingCurve.sol

```diff
@@ -12,6 +12,8 @@ contract BondingCurve {
     using SafeMath for uint256;
     address payable public owner;
 
+    bool public halted;
```

Comment:
Is it used by other contracts yet?

Reply:
I was going to use it for buyout, will remove